R/templates.R

```diff
@@ -33,6 +33,8 @@ exports_h <- function(declarations, wrappers) {
 // Generated by using torchexport::export() -> do not edit by hand
 #pragma once
 
+# include "common.h"
```

Comment:
We might want to make this conditional. Ie, only add if `common.h` exists somewhere in the directory.